### ISSUE TITLE
当使用AntDesignVueResolver({ importStyle: 'less' })  message组件没有样式

### ISSUE THREAD

**Issue by PoliWen** [antfu/unplugin-vue-components#443]
``` javascript
AntDesignVueResolver({ importStyle: 'less' })
```
``` javascript
import { message } from 'ant-design-vue'
```

**Comment by nabaonan** [comment]
https://github.com/antfu/unplugin-vue-components/issues/162   不在模板里写，这个插件检测不到

**Comment by Anxure** [comment]
如果你使用的 Vite，你可以使用 [unplugin-vue-components](https://github.com/antfu/unplugin-vue-components) 来进行按需加载。但是此插件无法处理非组件模块，如 message，这种组件需要手动加载
`import { message } from 'ant-design-vue';`
`import 'ant-design-vue/es/message/style/css'; //vite只能用 ant-design-vue/es 而非 ant-design-vue/lib`

**Comment by pjcollazo** [comment]
This issue is considered resolved then yes?

**Comment by voladelta** [comment]
> This issue is considered resolved then yes?

yes, it's on the docs, [getting-started#Import-on-Demand](https://www.antdv.com/docs/vue/getting-started#Import-on-Demand)
